src/main/webapp/js/userMapData.json

```diff
@@ -585,6 +625,7 @@
 ["N2N Services","USA"],
 ["Nash Rocky Mount Public Schools","USA"],
 ["Navarro College","USA"],
+["NC A&T State University","USA"],
 ["NC State University","USA"],
```

Comment:
These are different - North Carolina Agricultural and Technical State University is in Greensboro, and NC State University is in Raleigh.